server/views/patterns/typography/typography.config.json

```diff
@@ -0,0 +1,190 @@
+{
+  "context": {
+    "fonts": [
+      {
+        "font__name": "WB1",
```

Comment:
Doesn't need to be, was just matching the class name of the element they populated